src/frontend/components/MainCard/MainCard.js

```diff
@@ -1,6 +1,35 @@
-import { Navigate } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { Navigate, useNavigate } from "react-router-dom";
+import { useAuth } from "../../context/authContext";
+import { useCart } from "../../context/cartContext";
+import { addToCartService } from "../../services";
 import "./MainCard.css";
 const MainCard = ({ products }) => {
+  const navigate = useNavigate();
+  const { auth } = useAuth();
+  const { cart, setCart } = useCart();
+  const [inCart,setInCart]=useState(false)
+  useEffect(()=>{
+ cart.cartProducts.find((prod)=>prod._id === products._id) && setInCart(true)
+ 
+  
+
+  },[cart.cartProducts])
+
+  const addToCart = async() => {
+    console.log("add to cart")
```

Comment:
We should remove the console.log

Reply:
Thanks for mentioning...Will surely take care of it.